code/data_structures/src/stack/balanced_expression/balanced_expression.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <stack>
+
+using namespace std;
```

Comment:
Just remove usage of `using namespace std`, and you're good to go!

Reply:
@arnavb @AdiChat Done.